File: MachineProblems/Code/MP9.java
import java.util.Scanner;

public class MP9 {
    public static void main(String[] args) {
        Scanner myobj = new Scanner(System.in);
        int find = myobj.nextInt();
        int pos = 0;
        boolean found = false;
        int prev = 0;
        int holder;
        int numContainer = 0;

        for (int i = 0; numContainer <= find; i++) {
            
            if(numContainer == 0){
                prev = numContainer;
                numContainer++;
                continue;
            }

            if(numContainer == find){
                pos = i;
                found = true;
                break;
            }
            holder = numContainer;
            numContainer += prev;
            prev = holder;

        }
        if (found) {
            System.out.println( String.valueOf(find) + " is found on position " + String.valueOf(pos));
        } else {
            System.out.println( String.valueOf(find) + " does not exist");
        }


    }
}

File: MachineProblems/Code/MP9.py

number = int(input())

a, b = 0, 1
index = 0

if number == 0:
    print(f"{number} exists at position {index}")
else:
    while a <= number:
        if a == number:
            print(f"{number} exists at position {index}")
            break
        a, b = b, a + b
        index += 1
    else:
        print(f"{number} does not exist")
